### PR TITLE
New package: usbimager-1.0.10

### PR DESCRIPTION
**Pull request by filiprojek**
Closes: #42758

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/usbimager/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'usbimager'
+pkgname=usbimager
+version=1.0.10
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libX11-devel"
```
Comment (Luciogi) [inline srcpkgs/usbimager/template:6]: It should be ` makedepends=`, not `hostmakedepends=`

**Comment by Luciogi** [comment]
`build_style=gnu-makefile`
when template mentions , `build_style` variable, then you dont have have to declare functions like `do_build` and `do_install`
xbps-src will do build and install automatically

--- if it fails to build automatically, then you can write custom functions for build and install


If I said anything correct me reviewers

File: srcpkgs/usbimager/template (hunk 1 above)
Comment (filiprojek) [reply srcpkgs/usbimager/template:7]: if I make this change, the package can no longer be compiled for any architecture

**Comment by filiprojek** [comment]
When I try to build the package using ` build_style=gnu-makefile`, the package reports an error about missing libraries.
There's some problem with the vendored libraries. @Duncaen  advised me on irc to try to link system libraries instead of using vendored ones

I managed to build a package for `x86_64-musl` on my local machine. I have no idea why the package reports an error here.

I have not been able to build a package for these architectures:
 - `aarch64`
 - `armv7l`
 - `armv6l-musl`
 - `aarch64-musl`

On `aarch64` there is this error:
`/usr/bin/aarch64-linux-gnu-strip: Unable to recognise the format of the input file '/destdir/aarch64-linux-gnu/usbimager-1.0.10/usr/bin/usbimager'` 
@classabbyamp advised me that Makefile probably ignores cflags

Until I can fix them, I have masked them in the template.

**Pull request closed without merging**